lib/styleSizeTypes.js

```diff
@@ -0,0 +1,44 @@
+const { parse: mqifyParse } = require('mqify')
+const classPostfix = require('postcss-class-postfix')
+const postcss = require('postcss')
+
+const stylesheetTypeToToken = {
+  notsmall: ['ns', 'm'],
+  small: ['s'],
+  large: ['l'],
+  xlarge: ['xl']
+}
+
+module.exports = (stylesheetType, originalModules, mediaQueries) => {
```

Comment:
you could name this function before exporting it, as to make it clear its purpose
(and perhaps also comment it if needed)